modules/_labs/menu/react/lib/Menu.tsx

```diff
@@ -89,16 +90,22 @@ export default class Menu extends React.Component<MenuProps, MenuState> {
     if (this.props.isOpen && !prevProps.isOpen) {
       this.setInitialSelectedItem();
     }
-    if (this.props.isOpen && this.menuRef.current) {
-      this.menuRef.current.focus();
-    }
+    this.animateId = requestAnimationFrame(() => {
+      if (this.props.isOpen && this.menuRef.current) {
+        this.menuRef.current.focus();
+      }
+    });
   }
 
   componentDidMount() {
     this.setFirstCharacters();
     this.setInitialSelectedItem();
   }
 
+  componentWillUnmount() {
+    cancelAnimationFrame(this.animateId);
```

Comment:
Where is `cancelAnimationFrame` defined in this file?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Window/cancelAnimationFrame

Reply:
oh it's on the window. Geeez. 🤦